resdk/tests/unit/test_collections.py

```diff
@@ -29,7 +29,8 @@ def test_init(self, collection_mock):
 
     @patch('resdk.resources.collection.BaseCollection', spec=True)
     def test_data_types(self, collection_mock):
-        api_mock = MagicMock(**{'data.return_value': MagicMock(**{'get.return_value': DATA_SAMPLE[0]})})
+        api_mock = MagicMock(**{'get.return_value': DATA_SAMPLE[0]})
```

Comment:
I think using `api_mock` variable name twice is unnecessarily confusing. Can you change one name to something else?